encrypted-media-respec.html

```diff
@@ -2326,7 +2325,7 @@
           <ol>
             <li><p>Let the <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
             <li><p>Let <var>expiration time</var> be <code>NaN</code>.</p></li>
-            <li><p>If the new expiration time is not <code>NaN</code>, let <var>expiration time</var> be the new expiration time in milliseconds since 01 January 1970 UTC.</p></li>
+            <li><p>If the new expiration time is not <code>NaN</code>, let <var>expiration time</var> be that expiration time.</p></li>
```

Comment:
This is perhaps the only place where we really need to reference Time since it is the only place where it is visible. However, it doesn't hurt to do this any place that we get a time.

If we are going to include a reference at line 1813, we also need to do so in:
* the `update()` algorithm
* the Monitor for CDM State Changes algorithm